aws/table_aws_cloudfront_distribution.go

```diff
@@ -0,0 +1,329 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudfront"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCloudfrontDistribution(_ context.Context) *plugin.Table {
```

Comment:
```suggestion
func tableAwsCloudFrontDistribution(_ context.Context) *plugin.Table {
```
Please change wherever this function is used.